jenkins/build-pod.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Pod
+spec:
+  containers:
+  - name: node
+    image: node:12.16.1
```

Comment:
It's not necessary to change that I'm just wondering if it would be possible to dynamically get the node version number from `.nvmrc` file which we have in all projects?

Reply:
The whole point here is to use prebuild the existing image of node, for optimization. You also need to define the image version before the workflow executed

Reply:
OK, so that image has to be prebuilt and placed somewhere before using it. My question background was: what will be the way in the future to change the NodeJS version (we need to do that for all UI projects)? It will happen for sure sometime in the future, not soon of course as we did recently change from 10.x to 12.x. During my assignment in Cloudify I did it 2 or 3 times already. Please have a look on "Upgrading Node.js" article in R&D wiki space for details on how I understand the upgrade process. If you could update the description or add a comment there, I'd be grateful.